### DISCUSSION TITLE
Font size issues resulting in table misaligment & a possible bug report

### DISCUSSION THREAD

**Discussion by gramapp** [jlai/fitness-dashboard#10] [General]
![Fitbit screen shot](https://github.com/jlai/fitness-dashboard/assets/175133758/5f4f9e16-ef31-4891-ada4-1eaaa9bff797)

On second line due to longer text in the amount field the line of the table gets offset. If I reduce the zoom to 90% in the browser its ok. Not a big deal, but is there a way for nutrition dashboard to do this by itself?

Also how the elevation protein bar, did not pull in macronutrients. If I go in to the app and select nutritional info for this entry it shows macronutirents.

**Comment by jlai** [comment]
Hmm, it seems like the Fitbit API is not returning nutrition info for certain foods. It does get included in the total in the mobile app and fitbit.com/foods/log. That could be problematic.

Have a fix for the table misalignment that will go out later.

**Reply by gramapp** [reply]
The addition of "Not available due to Fitbit API limitations" in the table is a nice touch!
